synphot/spectrum.py

```diff
@@ -778,6 +778,67 @@ def plot(self, wavelengths=None, **kwargs):  # pragma: no cover
         w, y = self._get_arrays(wavelengths)
         self._do_plot(w, y, **kwargs)
 
+    @classmethod
+    def from_Spectrum1D(cls, spec, keep_neg=False):
+        """Create a spectrum from `specutils.Spectrum1D` object.
+
+        Parameters
+        ----------
+        spec : `specutils.Spectrum1D`
+
+        keep_neg : bool
+            See `~synphot.models.Empirical1D`.
+
+        Returns
+        -------
+        sp : `BaseSourceSpectrum`
+            Empirical spectrum.
+
+        """
+        # Spectrum1D is designed to be immutable, so no need to make
+        # copies of spectral_axis nor flux.
+        return cls(Empirical1D, points=spec.spectral_axis,
+                   lookup_table=spec.flux, keep_neg=keep_neg,
+                   meta={'header': spec.meta.copy()})
+
+    def to_Spectrum1D(self, wavelengths=None, **kwargs):
```

Comment:
I'm not sure I like the look of this where it has a weird mix of casing conventions.  What about:
```suggestion
    def to_spectrum1d(self, wavelengths=None, **kwargs):
```
?  Or even:
```suggestion
    def to_specutils(self, wavelengths=None, **kwargs):
```
?

Reply:
I debated with myself about this. The class is technically `Spectrum1D` with uppercase letters but that is also against PEP 8 naming convention for method names. Since you prefer all lowercase, I'll make that change.

However, I do not think it should be `to_specutils`. What if you have a different class in `specutils` that you wish to tie in to `synphot` in the future? Do you want this method to handle all those cases?

Reply:
Renamed to lowercase.